### PR TITLE
Finished CSS for reviews component

### PR DESCRIPTION
**Pull request by NgoMike**
https://trello.com/c/XOsmUQrG/8-m-complete-css

### CODE REVIEW

**Comment by NgoMike** [comment]
reviewscomponent

File: react/src/overallRatings.jsx [hunk 1]
```diff
@@ -4,11 +4,30 @@ const OverallRatings = (props) => {
...
+            </span>{restaurant.restaurantName} Ratings and Reviews 
+            <div className='overallrating'>
+              <span className='graph'>
+                <ul className='bargraph'>
+                  5<li class="reddeep"></li>
```
Comment (tmkuba) [inline react/src/overallRatings.jsx]: should these be "className" instead of "class"?

**Review by tmkuba — COMMENTED** [review]
one q about class vs className in one file, otherwise looks good!

**Review by matgbx — APPROVED** [review]
This looks great!

File: react/src/overallRatings.jsx (hunk 1 above)
Comment (NgoMike) [reply react/src/overallRatings.jsx]: thanks Mark, it should be className. I made the adjustments

**Pull request merged**